common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -2453,6 +2453,12 @@
     },
     "sqlState" : "42K0K"
   },
+  "INVALID_ITERATE_LABEL_USAGE_FOR_COMPOUND" : {
```

Comment:
Can these error classes be put together with subclasses? like
INVALID_LABEL.ITERATE_FOR_COMPOUND / INVALID_LABEL.IN_STATEMENT

Reply:
Done.